integration_test/third_party_apps_test/applications/couchbase/enable

```diff
@@ -13,13 +13,15 @@ fi
 sudo cp /etc/google-cloud-ops-agent/config.yaml /etc/google-cloud-ops-agent/config.yaml.bak
 
 # Configure the Ops Agent.
-sudo tee /etc/google-cloud-ops-agent/config.yaml > /dev/null << EOF
+sudo tee /etc/google-cloud-ops-agent/config.yaml > /dev/null << \EOF
```

Comment:
Why was the `\EOF` added in every enable script here?

Reply:
Adding \ so that the provider identifier is not interpreted as a shell variable. 

This content is eventually stored as a shell script in the test VM: 
[![image](https://github.com/user-attachments/assets/7878b7e1-b8c0-4003-8156-2ef10ac71843)](https://screenshot.googleplex.com/B5PWySZ5ciMRxP3)

Reply:
> Adding \ so that the provider identifier is not interpreted as a shell variable.

This does not explain why `EOF` has a `\`, I'm still confused about this.